morpho/plot/histo.py

```diff
@@ -339,9 +370,84 @@ def histo2D(param_dict):
     else:
         figurefullpath += '.pdf'
     can.SaveAs(figurefullpath)
+    return
 
+def histo2D(param_dict):
+    '''
+    Plot 2D histogram
+    '''
+    can, title, nbins_x, nbins_y = _prepare_canvas_2d(param_dict)
+    myfile = ROOT.TFile(param_dict['input_file_name'],"READ")
+    list_dataX, list_dataY, namedata = _get_root_data_2d(param_dict, myfile)
+    if(list_dataX is None):
+        return
+
+    histo = _get2Dhisto(list_dataX, list_dataY, [nbins_x,nbins_y], [0,0], title)
+    histo.GetXaxis().SetTitle(namedata[0])
+    histo.GetYaxis().SetTitle(namedata[1])
+
+    if 'root_plot_option' in param_dict:
+        histo.Draw(param_dict['root_plot_option'])
+    else:
+        histo.Draw('contz')
+    _save_histo(param_dict, title, "", can)
     return can
 
+def histo2D_divergence(param_dict):
+    '''
+    Plot 2D histogram
+    '''
+    can, title, nbins_x, nbins_y = _prepare_canvas_2d(param_dict)
+    myfile = ROOT.TFile(param_dict['input_file_name'],"READ")
+    list_dataX_div0, list_dataY_div0, \
+        list_dataX_div1, list_dataY_div1, \
+        namedata = _get_root_data_2d_divergence(param_dict, myfile)
+
+    if(list_dataX_div0 is None):
+        return
+
+    xmin = min(list_dataX_div0+list_dataX_div1)
```

Comment:
shouldn't this be xmin = min(list_dataX_div0, list_dataX_div1) instead of xmin = min(list_dataX_div0+list_dataX_div1)? ( comma instead of +)
same below?